app/models/logical_order_base.rb

```diff
@@ -26,7 +26,6 @@ def order=(order) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     self.nodes = graph.subjects.reject { |x| x == rdf_subject }
     @order = nil
     @logical_order = nil
-    order
```

Comment:
Huh, did Lint say to drop the return value here?  (I thought it the ruby way to return the assigned value after successful assignment.)